modules/oe_theme_helper/templates/oe-theme-helper-call-label-status.html.twig

```diff
@@ -1,9 +1,11 @@
 {#
 /**
  * @file
- * Theme override for the status extra field on Call for tenders content type.
+ * Theme override for the status extra field as label.
  *
- * @see \Drupal\oe_theme_content_call_tenders\Plugin\ExtraField\Display\CallForTendersLabelStatusExtraField::viewElements()
+ * @see template_preprocess_oe_theme_helper_in_page_navigation()
```

Comment:
Ups.. but since we don’t have a template_preprocess_oe_theme_helper_call_label_status (its not needed) I investigated if there would be a generic preprocess that runs (that I should point to, I expected there would be), but low and behold nothing of sorts, so I just removed that line.